Dockerfile

```diff
@@ -27,6 +28,9 @@ RUN npm install
 
 # Stage 2: Builder
 FROM base AS builder
+
+RUN apk update && apk upgrade --no-cache && apk add --no-cache bash vim
```

Comment:
I wonder if we should document somewhere that vim is available in the containers?